webapp/utils/utils.jsx

```diff
@@ -314,14 +313,9 @@ export function getTimestamp() {
 }
 
 // extracts links not styled by Markdown
-export function extractLinks(text) {
-    text; // eslint-disable-line no-unused-expressions
-    Autolinker; // eslint-disable-line no-unused-expressions
-
-    // skip this operation because autolinker is having issues
-    return [];
-
-    /*const links = [];
+export function extractFirstLink(text) {
+    const pattern = /(^|[\s\n]|<br\/?>)((?:https?|ftp):\/\/[\-A-Z0-9+\u0026\u2019@#\/%?=()~_|!:,.;]*[\-A-Z0-9+\u0026@#\/%=~()_|])/gi;
```

Comment:
I don't know if I trust this. Haven't we had a lot of problems in the past? @hmhealey